src/test/ui/generics/wrong-number-of-args.stderr

```diff
@@ -112,13 +112,9 @@ note: struct defined here, with 2 type parameters: `A`, `B`
    |
 LL |     struct Ty<A, B>;
    |            ^^ -  -
-help: use angle brackets to add missing type arguments
-   |
-LL |     type A = Ty<A, B>;
-   |                ^^^^^^
```

Comment:
Losing these seems inconvenient to me.

Reply:
I fixed this, but it isn't perfect yet. We currently get:

```
error[E0106]: missing lifetime specifier
  --> src/test/ui/generics/wrong-number-of-args.rs:43:14
   |
43 |     type A = Ty;
   |              ^^ expected named lifetime parameter
   |
help: consider introducing a named lifetime parameter
   |
43 |     type A<'a> = Ty<'a>;
   |           ^^^^   ^^^^^^
```

and another error suggestion for the missing type argument:

```
error[E0107]: this struct takes 1 type argument but 0 type arguments were supplied
  --> src/test/ui/generics/wrong-number-of-args.rs:43:14
   |
43 |     type A = Ty;
   |              ^^ expected 1 type argument
   |
note: struct defined here, with 1 type parameter: `T`
  --> src/test/ui/generics/wrong-number-of-args.rs:41:12
   |
41 |     struct Ty<'a, T>;
   |            ^^     -
help: add missing type argument
   |
43 |     type A = Ty<T>;
   |              ^^^^^
```

Once either suggestion is applied, the correct suggest follows in the next check.

Reply:
These are included again.